README.md

```diff
@@ -165,5 +195,24 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+
+3wishes = Lamp(wish1:'', wish2:'', wish3:'', genie: Genie)
```

Comment:
careful of your syntax here! I believe the question was asking to set a `wishes_remaining` property on a `Genie` to `three` upon creation, i.e.:

```ruby
genie = Genie.create(wishes_remaining: 3, name: "Genie")
```